extension/content_script.js

```diff
@@ -424,43 +413,34 @@ function setupKeyListeners() {
 
 		switch (evt.code) {
 			case 'ArrowLeft': // Move back
-				stopFadeTracker();
-                startMove(direction.BACKWARD);
+				stopMove();
```

Comment:
stopMove() should handle all the animation woes later. So, no need to call stopfadetracker